docker/base/centos8/install-mpi.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
@SteVwonder: it seems this is the exact copy of the script used for Centos7. In terms of code management, we can create a `docker/base/scripts` and add the commonly used scripts like this there? That way, if a bug needs to be fixed in the future, we just change it in one place?

Reply:
Yeah, I can do that.  There will need to be some special casing within the script still for Centos7 vs 8 since 7 requires extra packages (slurm-pmi) to be installed and then removed, but it will definitely cut down on the duplication.

I will also need to tweak the way we build the docker images to make sure the `base` directory is the top-level directory of the build context as oppsed to `base/centos7` or `base/centos8` [(link for more context on what I'm talking about)](https://stackoverflow.com/questions/39376786/docker-and-symlinks#39382248). But that is definitely doable.